invenio_rdm_records/vocabularies/convert_to_new_vocabulary.py

```diff
@@ -69,31 +68,46 @@ def to_dict(self, csv_row):
             "props": dict(csv_row)
         }
 
-    @property
-    def csv_filename(self):
-        """Get csv filename."""
-        return Path(self.base_filename).with_suffix(".csv")
 
-    @property
-    def yaml_filename(self):
-        """Get yaml filename."""
-        return Path(self.base_filename).with_suffix(".yaml")
+class Converter:
+    def __init__(self, converter_type):
+        self.converter_type = converter_type
+
+    def convert(self, filepath, dirpath):
+        """Write converted filepath file to dirpath directory."""
+        yaml_filepath = dirpath / filepath.with_suffix(".yaml").name
+        with open(filepath) as csv_file:
+            reader = csv.DictReader(csv_file)
+            reader = hierarchized_rows(reader)
+            with open(yaml_filepath, "w") as yaml_file:
+                yaml.dump(
+                    [self.converter_type.to_dict(row) for row in reader],
+                    yaml_file
+                )
+            click.secho("Conversion done!", fg="green")
+
+
+@click.command()
+@click.argument('csv_filename')
+@click.option('--to', default='.', help='Export destination directory.')
+def convert(csv_filename, to):
```

Comment:
moderate: should we move this to the `cli.py`?

Reply:
I was inclined not to since it's like those upgrade scripts: meant to be used for this release and that's it. Also the command would be quite long: `pipenv run invenio rdm-records fixtures convert <filename> <dirname>` ...